virttest/utils_test/libvirt.py

```diff
@@ -3760,3 +3761,354 @@ def check_dumpxml(vm, content, err_ignore=False):
                 raise exceptions.TestFail("Expected '%s' was not found in "
                                           "%s's xml" % (content, vm.name))
     return True
+
+
+#cgroup related functions
+class CgroupTest(object):
+
+    """Class for cgroup related test"""
+
+    __vm_pid = ""
+    __virsh_blkiotune_output_mapping = {"weight": "weight",
+                                        "device_read_iops_sec": "riops",
+                                        "device_write_iops_sec": "wiops",
+                                        "device_read_bytes_sec": "rbps",
+                                        "device_write_bytes_sec": "wbps"}
+    __cgroup_v1_blkio_file_mapping = {"weight": "blkio.bfq.weight",
```

Comment:
Please also consider different value, for example, for RHEL 7.x

Reply:
per we talked about this, cgroup2 can be tested only on rhel8, and rhel7 have existing cases, so we're not adding this job to rhel7 test scope. No need to deal with rhel7 issues for now.